### PR TITLE
Bielliptic 3D maneuvers validation

### PR DESCRIPTION
**Pull request by jorgepiloto**
The last step before closing #10. Opening this as draft request since the Orekit API does not output expected results from GMAT although poliastro does.

### CODE REVIEW

**Comment by jorgepiloto** [comment]
Tests were passing locally. This seems to be related with https://github.com/poliastro/poliastro/issues/1100, which is being addressed in https://github.com/poliastro/poliastro/pull/1101.

**Comment by jorgepiloto** [comment]
Oh! I see why this still failing... Either we impose the numpy version or solve first #19, as the CI bug was fixed within the nightly build.

**Comment by astrojuanlu** [comment]
Ready for rebase :)

File: gmat/tests/gmat_validate_bielliptic3D.script [hunk 1]
```diff
@@ -0,0 +1,164 @@
+%--------------------------------------------
+%-- GMAT validate Bi-elliptic 3D maneuvers --
+%--------------------------------------------
+
+% ABOUT
+% -----
+%
+% The following script tries to validate
```
Comment (astrojuanlu) [inline gmat/tests/gmat_validate_bielliptic3D.script:8]:
```suggestion
% The following script is used to validate
```

**Review by astrojuanlu — APPROVED** [review]
Looks good to me! Ready for merge after the rebase if the tests pass 👍🏽

File: gmat/tests/gmat_validate_bielliptic3D.script (hunk 1 above)
Comment (jorgepiloto) [reply gmat/tests/gmat_validate_bielliptic3D.script]: Fixed :+1:

**Comment by jorgepiloto** [comment]
All tests passing! :tada:

**Pull request merged**